### DISCUSSION TITLE
Github Student Pack Benefits Are Not Activated

### DISCUSSION THREAD

**Discussion by dhruvgcoder** [community/community#147474] [GitHub Education] [answered]
### Hi everyone,

Hi everyone , 
I have applied for github student developer pack on Dec 16 , 2024 and my request got approved within 5 mins , but its been 3 days and today is Dec 19 , 2024 and my pack benifits are still not activated yet , there is a message your benefits will be available soon , but its a long delay , should anyone can tell me how long i have to wait for this .....

**Comment by fz-g** [comment]
I'm in the same boat and hoping for a swift activation as well. Let's hope GitHub resolves this for us soon!

**Reply by Sev6n** [reply]
Same!

**Reply by dhruvgcoder** [reply]
DONE ! Activated Now ...

**Reply by Sev6n** [reply]
@dhruvgcoder Did you get activated?

**Reply by dhruvgcoder** [reply]
yes , issue solved

**Reply by Sev6n** [reply]
Great! Still waiting for mine to get activated :/

**Reply by lrlunin** [reply]
Same, since 12th of December is not activated.

**Reply by Sev6n** [reply]
@lrlunin Hopefully someone working for GitHub notices this discussion thread and can get all of us activated soon. 👍

**Reply by lrlunin** [reply]
@Sev6n Nice profile picture, by the way ❤️

**Reply by Sev6n** [reply]
@lrlunin Thanks!! I’m also sure this is a manual process with the time it’s currently taking, I hope you get activated soon!

**Reply by Jackstark20** [reply]
i am still  not activate . why ?
![屏幕截图 2024-12-19 215950](https://github.com/user-attachments/assets/e9b74583-73ce-4628-b818-6a6cf3cb5034)

**Reply by Sev6n** [reply]
@Jackstark20 Please read the highlighted [answer](https://github.com/orgs/community/discussions/147474#discussioncomment-11614624) I posted. :)

**Reply by samus-aran** [reply]
@fz-g - You're approved - you should have access to your academic benefits 🎉 
@Sev6n - You are still within your 72 hour transition period. Wait a little longer. ⏲️ 
@lrlunin - Did you submit your application on the 12th? or the 16th? If it was the 16th then its still within your 72 hour transition period. 
@Jackstark20 - You are still within your 72 hour transition period. Wait a little longer. ⏲️

**Reply by lrlunin** [reply]
@samus-aran Hey,
it does not work somehow. I also receive plenty of GitHub E-Mails. I guess there is some global issue with Github Educational. 
![image](https://github.com/user-attachments/assets/51a22f3c-d119-4b3a-881f-b3a00d78b065)

UPD: received an E-Mail minutes ago, thatnks for your effort

**Comment by Sev6n** [comment, marked as answer]
Okay, I found the answer. According to most of my friends who applied for the student program and got accepted, it takes around 3-5 days for your features to activate. In this case, you should wait an extra day or a couple more hours and you should be activated. If your features are still not activated after this, you might want to read [this](https://github.com/orgs/community/discussions/111352).
